### DISCUSSION TITLE
APK API Level

### DISCUSSION THREAD

**Discussion by outof7** [airsdk/Adobe-Runtime-Support#1248] [Q&A]
Dear all,

When trying to upload my APK that has been compiled with the latest Harman sdk, I am getting the following error message from Google Play Console:

_Your app currently targets API level 27 and must target at least API level 29 to ensure that it is built on the latest APIs optimised for security and performance. Change your app's target API level to at least 29._

Google search did not return anything conclusive!

Thank you for any pointer,

Alain

**Comment by marchbold** [comment]
The target sdk is specified in your manifest additions, you may have inadvertently set it to an older version. Look for something like this in your manifest additions:

```xml
<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="27" />
```

And change the `27` to at least `29`
